mozdownload/scraper.py

```diff
@@ -862,7 +862,7 @@ def platform_regex(self):
 
         PLATFORM_FRAGMENTS = {'linux': 'linux',
                               'linux64': 'linux64',
-                              'mac': 'macosx',
+                              'mac': r'macosx(64)?',
```

Comment:
As mentioned in my last review, the brackets are not necessary. That does not only apply to `mac64` but also `mac`.